backend/loi-service/src/main/java/org/egov/works/util/LOIConstants.java

```diff
@@ -0,0 +1,6 @@
+package org.egov.works.util;
+
+public class LOIConstants {
+    public static final String MDMS_MODULE_NAME = "tenant";
```

Comment:
Rename to MDMS_TENANT_MODULE_NAME. Same apples to the other variable. 
There might be other module names for which we could be calling mdms from the loi-service.

Reply:
Done.